File: Composicion y Agregacion/Ejercicio8/Java/Empleado.java
import java.util.ArrayList;
class Empleadoo {
    public String nombre;
    public String puesto;
    public double salario;
    public Empleadoo(String nombre, String puesto, double salario) {
        this.nombre = nombre;
        this.puesto = puesto;
        this.salario = salario;
    }
    public void mostrar() {
        System.out.println("Empleadoo: " + nombre + ", Puesto: " + puesto + ", Salario: BOB" + salario);
    }
}
class EquipoTrabajo {
    public String nombre;
    public ArrayList<Empleadoo> empleados;
    public EquipoTrabajo(String nombre) {
        this.nombre = nombre;
        this.empleados = new ArrayList<>();
    }
    public void agregar(Empleadoo e) {
        empleados.add(e);
    }
    public void mostrar() {
        System.out.println("Equipo: " + nombre);
        for (Empleadoo e : empleados) {
            e.mostrar();
        }
    }
}
public class Empleado {
    public static void main(String[] args) {
        EquipoTrabajo e = new EquipoTrabajo("Desarrollo");
        e.agregar(new Empleadoo("Sofia", "Programadora", 1200));
        e.agregar(new Empleadoo("Carlos", "Analista", 1100));
        e.mostrar();
    }
}

File: Composicion y Agregacion/Ejercicio8/Python/Empleado.py
class Empleado:
    def __init__(self, nombre, puesto, salario):
        self.nombre = nombre
        self.puesto = puesto
        self.salario = salario
    def mostrar(self):
        print("Nombre:", self.nombre)
        print("Puesto:", self.puesto)
        print("Salario:", self.salario)
class EquipoTrabajo:
    def __init__(self, nombre):
        self.nombre = nombre
        self.empleados = []
    def agregar(self, empleado):
        self.empleados.append(empleado)
    def mostrar(self):
        print("Equipo de Trabajo:", self.nombre)
        for i in self.empleados:
            i.mostrar()
e = EquipoTrabajo("Proyecto A")
e.agregar(Empleado("Maria", "Diseñadora", 3500))
e.agregar(Empleado("José", "Programador", 4000))
e.mostrar()
